tv_apps/smart-system/js/permission_manager.js

```diff
@@ -331,28 +332,41 @@
 
       var message = '';
       var permissionID = 'perm-' + this.permissionType.replace(':', '-');
-      var _ = navigator.mozL10n.get;
 
       if (detail.isApp) { // App
         var app = applications.getByManifestURL(detail.manifestURL);
-        message = _(permissionID + '-appRequest',
-          { 'app': new ManifestHelper(app.manifest).name });
+        message = {
+          id: permissionID + '-appRequest',
+          args: { 'app': new ManifestHelper(app.manifest).name }
+        }
 
-        this.title.innerHTML = _('title-app');
+        this.title.setAttribute( 'data-l10n-id', 'title-app');
         if (this.isCamSelector) {
-          this.title.innerHTML = _('title-cam');
+          this.title.setAttribute( 'data-l10n-id', 'title-cam');
         }
-        this.deviceSelector.innerHTML = _('perm-camera-selector-appRequest',
-            { 'app': new ManifestHelper(app.manifest).name });
+        navigator.mozL10n.setAttributes(
+          this.deviceSelector,
+          'perm-camera-selector-appRequest',
+          { 'app': new ManifestHelper(app.manifest).name }
+        );
       } else { // Web content
-        message = _(permissionID + '-webRequest', { 'site': detail.origin });
+        message = {
```

Comment:
Comparing to [1], they didn't use setAttributes on `message` variable. I feel you are correct here. May you help to confirm this one and file a bug for phone's System app?

[1] https://github.com/mozilla-b2g/gaia/commit/52ee3c33aacd17ba32de3e187ecfc554ef127993